en/app.ftl

```diff
@@ -72,7 +72,7 @@ nav-settings = Settings
 nav-support = Help and Support
 nav-sign-out = Sign Out
 nav-contact = Contact Us
- 
+
```

Comment:
this change seems to introduce an error? See the commit with an `x` status (the last of a series of commits). Is it the line break breaks the section under `## Mobile menu text`? Not sure.